### DISCUSSION TITLE
Credentials

### DISCUSSION THREAD

**Discussion by Bdiwy** [4xmen/xshop#40] [Q&A] [answered]
Hello, I have installed XShop. What is the email and password to start using it?

**Comment by YasinDehfuli** [comment, marked as answer]
Hello dear @Bdiwy, as you see in readme.md & database. 

Email: admin@example.com
Pass: password

**Reply by mmdreza2003** [reply]
please learn me to how installation tnx🙏

**Reply by Bdiwy** [reply]
Maybe i should contribute in adding the installation process in the readme file

**Reply by YasinDehfuli** [reply]
Of course, no problem.

**Reply by mmdreza2003** [reply]
thank you 🙏
